plutus-conformance/README.md

```diff
@@ -6,28 +6,48 @@ This package aims to provide an official and comprehensive test suite that check
 
 ## Specification
 
-The tests currently covers or will cover:
+The tests currently cover or will cover the Haskell and Agda implementation of:
 
-- An evaluation test suite for UPLC (`uplc-eval-test`)
-- A typechecking test suite for TPLC, including checking of alpha equivalence. (`tplc-typecheck-test`)
-- An evaluation test suite for TPLC (`tplc-eval-test`)
-- A TPLC to UPLC erasure test suite
-- Costing conformance 
+- UPLC evaluation
+- Typechecking for TPLC, including checking of alpha equivalence. (`tplc-typecheck-test`)
+- TPLC evaluation
+- Erasure of TPLC to UPLC
 - Coverage test
+<!-- - Costing conformance? -->
```

Comment:
Ooh yes, we should have a ticket for this, I think we don't at the moment. Can you make one?

Reply:
[Done.](https://input-output.atlassian.net/browse/PLT-514)